integration.py

```diff
@@ -78,11 +77,8 @@ def get_text(self, resp):
         else:
             return resp.data.decode('utf-8')
 
-    def validate_sample(self, namespace, name, messages):
+    def validate_sample(self, namespace, doc_type, doc_version, messages):
         start = time.time()
-        submission, doc_type, doc_version = (
```

Comment:
This is the wrong place to be extracting information from the filename, I moved this out into the main portion of the reporter routine.